neuron/src/app/Neuron/CLI/Types.hs

```diff
@@ -127,13 +127,13 @@ commandParser defaultNotesDir today = do
               <> long "format"
               <> help "The document format of the new zettel"
       edit <- switch (long "edit" <> short 'e' <> help "Open the newly-created zettel in $EDITOR")
-      day <-
-        option dayReader $
-          long "day"
-            <> metavar "DAY"
-            <> value today
-            <> showDefault
-            <> help "Zettel creation date in UTC"
+      dateParam <-
+        option dateReader $
+          long "date"
+            <> metavar "DATE"
+            <> value (Left today)
```

Comment:
The default should be `Right now`, I think? I imagine we are making `neuron new` inject the full datetime by default in the generated zettel note.